README.md

```diff
@@ -25,9 +25,23 @@ For debugging like setup you'll need (but since it's used for internal tools, th
 
 `get()` and `getone()` wrap the `dispatch()` method and yield results as they are submitted in the web form, providing an interface that resembles the `parse_args()` method.
 
+### Dependencies ###
+
+`argparseweb` requires `web.py` to be available. You can install it (check for the latest version) by:
+
+```bash
+wget http://webpy.org/static/web.py-0.38.tar.gz
+tar -xvzf web.py-0.38.tar.gz
+cd web.py-0.38
+python setup.py install
+```

```

Comment:
Isn't `pip install web.py` easier?

Thanks for taking the time to make usage clear!